js/main.js

```diff
@@ -146,3 +203,9 @@ app.on('activate', function () {
         windowMgr.showMainWindow();
     }
 });
+
+app.setAsDefaultProtocolClient('symphony');
+
+app.on('open-url', function (event, url) {
+    handleProtocolAction(url);
```

Comment:
also noticed in documentation here: https://electron.atom.io/docs/api/app/#event-open-url-macos that something might be needed in plist???